lib/chef-dk/command/provision.rb

```diff
@@ -0,0 +1,399 @@
+#
+# Copyright:: Copyright (c) 2014 Chef Software Inc.
+# License:: Apache License, Version 2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+require 'chef-dk/command/base'
+require 'chef-dk/configurable'
+require 'chef-dk/chef_runner'
+require 'chef-dk/policyfile_services/push'
+
+require 'chef/provisioning'
+
+module ChefDK
+
+  module ProvisioningData
+
+    def self.reset
+      @context = nil
+    end
+
+    def self.context
+      @context ||= Context.new
+    end
+
+    class Context
+
+      attr_accessor :action
+
+      attr_accessor :node_name
+
+      attr_accessor :enable_policyfile
+
+      attr_accessor :policy_group
+
+      attr_accessor :policy_name
+
+      attr_accessor :extra_chef_config
+
+      def initialize
+        @extra_chef_config = ""
+      end
+
+      def convergence_options
+        {
+          chef_server: Chef::Config.chef_server_url,
```

Comment:
I don't think this is necessary - chef-provisioning should default to this same value you have set here.  Did you find an issue with that in testing?